packages/story-editor/src/components/header/buttons/publish.js

```diff
@@ -109,44 +99,36 @@ function PublishButton({ forceIsSaving }) {
   }, [refreshPostEditURL, saveStory, hasFutureDate, title, canPublish]);
 
   const handlePublish = useCallback(() => {
-    if (
-      canPublish &&
-      (shouldReviewDialogBeSeen || isUpdatedPublishModalEnabled)
-    ) {
+    updateToAllowPriorityIssues();
```

Comment:
as soon as the publish button in the header is hit we want to allow priority issues to surface in the checklist.